frontend/app/module/election/page/ElectionStatusPage.tsx

```diff
@@ -1,22 +1,172 @@
+import { ReactNode, useMemo } from "react";
 import { Link, useNavigate } from "react-router-dom";
 
-import { ElectionStatusProgress } from "app/component/election/ElectionStatusProgress";
 import { Footer } from "app/component/footer/Footer";
 import { NavBar } from "app/component/navbar/NavBar";
 
-import { useElection, useElectionStatus } from "@kiesraad/api";
+import {
+  PollingStation,
+  PollingStationStatus,
+  PollingStationStatusEntry,
+  useElection,
+  useElectionStatus,
+} from "@kiesraad/api";
 import { t } from "@kiesraad/i18n";
-import { Alert, Button, PageTitle } from "@kiesraad/ui";
+import { IconDot, IconPlus } from "@kiesraad/icon";
+import {
+  Alert,
+  Badge,
+  Button,
+  Icon,
+  PageTitle,
+  PercentageAndColorClass,
+  Progress,
+  ProgressBar,
+  ProgressBarColorClass,
+  Table,
+} from "@kiesraad/ui";
+
+import cls from "./ElectionStatusPage.module.css";
+
+interface PollingStationWithStatus extends PollingStation, PollingStationStatusEntry {}
+
+const statusCategories = ["unfinished", "in_progress", "definitive", "not_started"] as const;
+type StatusCategory = (typeof statusCategories)[number];
+
+const categoryColorClass: Record<StatusCategory, ProgressBarColorClass> = {
+  unfinished: "unfinished",
+  in_progress: "in-progress",
+  definitive: "definitive",
+  not_started: "not-started",
+};
+
+const categoryToStatus: Record<StatusCategory, PollingStationStatus> = {
+  // TODO: future `second_entry_unfinished` status should be added to `unfinished`
+  //  future `second_entry_in_progress` status should be added to `in_progress`
+  unfinished: "first_entry_unfinished",
+  in_progress: "first_entry_in_progress",
+  definitive: "definitive",
+  not_started: "not_started",
+};
+
+function getTableHeaderForCategory(category: StatusCategory): ReactNode {
+  function CategoryHeader({ children }: { children?: ReactNode[] }) {
+    return (
+      <Table.Header key={category} backgroundStyling>
+        <Table.Column key={`${category}-number`} number>
+          {t("number")}
+        </Table.Column>
+        <Table.Column key={`${category}-name`}>{t("polling_station")}</Table.Column>
+        {children}
+      </Table.Header>
+    );
+  }
+  const typistColumn = <Table.Column key={`${category}-typist`}>{t("typist")}</Table.Column>;
```

Comment:
I thought we were not going to include this column until we had users, so we'd have something to put into the column?

Reply:
@jorisleker Do you prefer to hide the `Invoerder` column for now, or shall we leave it empty until we have user accounts?

Reply:
A third option would be to hard-code "Invoerder" as the value.

Reply:
Removed in 12c066a4625030ceb263d15c26a38d6c7d58875e